ui/client/components/uiComponents/Branding.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
No comment really needed here. Simple utility that's used to support every instance (except the `index.js` favicon switch) where we want to check the `COMPANY_BRANDING` environment variable. Two other small utility functions to translate the env variable into the formatted names, for use both in and outside of JSX.